playbooks/installation/index.adoc

```diff
@@ -120,13 +139,14 @@ openshift_master_api_port=443 <1>
 openshift_master_console_port=443 <2>
 openshift_portal_net=172.30.0.0/16 <3>
 osm_cluster_network_cidr=10.128.0.0/14 <4>
+osm_host_subnet_length=9 <5>
```

Comment:
Need to add this to all hostfile snippets in the doc so we don't lose it after this section.

Reply:
I have added all this, let me know if there are other outstanding issues